EduNLP/Vector/quesnet/quesnet.py

```diff
@@ -5,50 +5,58 @@
 
 
 class QuesNetModel(object):
-    def __init__(self, pretrained_dir, tokenizer=None, device="cpu"):
+    def __init__(self, pretrained_dir, img_dir=None, device="cpu", **argv):
```

Comment:
Use **kwargs instead of **argv. Check all code @nnnyt

Reply:
All the `**argv` have been replaced by `**kwargs` now